dimod/binary_quadratic_model.py

```diff
@@ -1755,6 +1755,10 @@ def to_serializable(self, use_bytes=False, bias_dtype=np.float32,
 
         return doc
 
+    def _asdict(self):
+        # support simplejson encoding
+        return self.to_serializable()
+
```

Comment:
Yes, that's it. Unfortunately that doesn't work for stdlib's `json` module, although `json` is based on `simplejson` (I wasn't aware they omitted some parts. Or maybe they just forked an older version of `simplejson`.. Don't know, will have to check.)